content/blog/2023-11-09.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "2023-11-09"
+date = 2023-11-09
++++
+
+---
+
+Today is Thursday, day 313 of 365 (85.75%). We're in week 45 of 52 (86.53%).
+
+## Planning
+
+What I am going to do today:
+
+### snippets API
+- [x] Work on the flutter client
+- [x] Create github actions yaml
+- [] Implement unit tests
+
+### snippets API
+- [x] Create github actions yaml
+- [x] Add pre-commit default actions and black linter to github actions
+- [] Implement unit tests
+- [] Add unit tests to github actions
+
+## Activities executed
+
+Today I I added two dropdowns with the style and language options to the Flutter client and added pre-commit basic configuration and the github-actions folder to it. I also added the gh-actions folder to the API.
+
+## Learnings
+
+Today I learned about the [DropdownButton class](https://api.flutter.dev/flutter/material/DropdownButton-class.html?v=1.0.22&gclid=Cj0KCQiAo7KqBhDhARIsAKhZ4uj_sxFdzwUVYsPfwzXsSkOBewspfYZ8JSd050ACIO-fINXfmvFo1SwaAkkOEALw_wcB&gclsrc=aw.ds) which allowed me to create the dropdowns with the language and style options from the API and feel like I'm getting better at handling Flutter's types, when I got any errors about that I was able to fix it easily.
+
+I also learned about [best practices when it comes to consuming APIs in Flutter](https://www.section.io/engineering-education/consuming-apis-in-flutter/).
```

Comment:
Please add the 2 other URLs I posted on slack. Or add a new `## Readings` section when you read articles I suggest.